test/logger-helper-test.js

```diff
@@ -1011,8 +1011,10 @@ describe('logger-helpers tests', function () {
                     body: 'body',
                     test1: 'test2'
                 };
-                response.headers = undefined;
-                response._headers = undefined;
+
+                Object.keys(response.getHeaders())
```

Comment:
there is no method on the object to remove all the header keys in batch without involving the internal properties like in the past.